tests/functional/spec/numbers.spec.js

```diff
@@ -94,5 +94,9 @@ describe("Number validation", () => {
 
       await expect(await browser.getUrl()).to.contain(SubmitPage.pageName);
     });
+
+    it("When a number with more than 3 decimal places has been entered, Then it should be displayed correctly on the summary", async () => {
+      await expect(await $(SubmitPage.testDecimal()).getText()).to.equal("£11.12345");
```

Comment:
I suggest updating one of the number to have > 3 trailing decimal zeros.

Reply:
Updated 👍